content/solutions/pipeline.adoc

```diff
@@ -29,8 +29,7 @@ features like:
 * checking the pipeline definition into source control (`Jenkinsfile`)
 * support for extending the domain specific language with additional,
   organization specific steps, via the
-  "link:https://www.cloudbees.com/blog/jenkins-workflow-using-global-library-implement-re-usable-function-call-secured-http-endpoint[global
-  library]" feature
```

Comment:
This link is there (just slightly different): 
https://www.cloudbees.com/blog/jenkins-workflow-using-global-library-implement-reusable-function-call-secured-http-endpoint